# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1427312321 -3600
#      Wed Mar 25 20:38:41 2015 +0100
# Node ID 62b73798a84f958acb0eec4ba63aa59e952ab58c
# Parent  643de08e524cd8c684ab7cc2491c9a0c697d46a6
* shortcut-manager.cc (do_init_data): new default shortcut for zoom normal (bug #44627)

diff --git a/libgui/src/shortcut-manager.cc b/libgui/src/shortcut-manager.cc
--- a/libgui/src/shortcut-manager.cc
+++ b/libgui/src/shortcut-manager.cc
@@ -287,17 +287,17 @@ shortcut_manager::do_init_data ()
         QKeySequence ());
   init (tr ("Zoom In"), "editor_view:zoom_in", QKeySequence::ZoomIn);
   init (tr ("Zoom Out"), "editor_view:zoom_out", QKeySequence::ZoomOut);
 #if defined (Q_OS_MAC)
   init (tr ("Zoom Normal"), "editor_view:zoom_normal",
         QKeySequence (ctrl + Qt::Key_Underscore));
 #else
   init (tr ("Zoom Normal"), "editor_view:zoom_normal",
-        QKeySequence (ctrl_alt + Qt::Key_0));
+        QKeySequence (ctrl + Qt::Key_Period));
 #endif
 
   // debug
   init (tr ("Toggle Breakpoint"), "editor_debug:toggle_breakpoint",
         QKeySequence ());
   init (tr ("Next Breakpoint"), "editor_debug:next_breakpoint",
         QKeySequence ());
   init (tr ("Previous Breakpoint"), "editor_debug:previous_breakpoint",
